test/compositemark/errorbar.test.ts

```diff
@@ -62,8 +62,8 @@ describe('normalizeErrorBar with raw data input', () => {
             y2: {field: 'upper_people', type: 'quantitative'},
             x: {field: 'age', type: 'ordinal'},
             tooltip: [
-              {field: 'upper_people', title: 'Upper error of people', type: 'quantitative'},
-              {field: 'lower_people', title: 'Lower error of people', type: 'quantitative'},
+              {field: 'upper_people', title: 'mean + stderr of people', type: 'quantitative'},
```

Comment:
We use Titlecase for standard title names.  (e.g., `"Mean of x"`)

Reply:
Would you prefer `Mean(people) + Stderr(people)` or `Mean + Stderr of people`?